src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversEmpty.tsx

```diff
@@ -82,8 +82,8 @@ export const ThorchainSaversEmpty = ({ assetId, onClick }: ThorchainSaversEmptyP
   }, [assetId, openFiatRamp])
 
   const needAssetTranslation: TextPropTypes['translation'] = useMemo(
-    () => ['common.needAsset', { asset: asset?.name }],
-    [asset?.name],
+    () => ['common.needAsset', { asset: isRunePool ? asset?.symbol : asset?.name }],
```

Comment:
I would almost just update everything to `symbol` instead of `name`. We are using symbol in the description as well so it feel like it would make more sense to use the same vernacular throughout imo. @shapeshift/product

Reply:
Pretty much the same comment as kev here - also this doesn't generalize and `THORChain` will be displayed instead of RUNE in a few other places (i.e asset rows in your screenshot displaying THORChain reward instead of RUNE)

Reply:
Used asset.symbol in [0ec463b](https://github.com/shapeshift/web/pull/7540/commits/0ec463b88d224c917700712c9b027ac53450b68b)